src/processTargets/modifiers/scopeHandlers/DocumentScopeHandler.ts

```diff
@@ -0,0 +1,59 @@
+import { Position, Range, TextEditor } from "vscode";
+import { Direction, ScopeType } from "../../../typings/targetDescriptor.types";
+import { getDocumentRange } from "../../../util/rangeUtils";
+import { DocumentTarget } from "../../targets";
+import { OutOfRangeError } from "../targetSequenceUtils";
+import NotHierarchicalScopeError from "./NotHierarchicalScopeError";
+import { TargetScope } from "./scope.types";
+import { ScopeHandler } from "./scopeHandler.types";
+
+export default class DocumentScopeHandler implements ScopeHandler {
+  public readonly scopeType = { type: "line" } as const;
```

Comment:
Shouldn't this be document and not line?

Reply:
Indeed. Nice catch

Reply:
Ok should be good to go now